R/model-options.R

```diff
@@ -33,3 +33,18 @@ get_model_options_template <- function(art, anc) {
   }
   templates
 }
+
+#' Validate a set of model options
+#'
+#' This validates that a set of model options can be used to run the model
+#'
+#' @param data The set of input data for the model run
+#' @param options Key-value list of model options
+#'
+#' @return TRUE if valid otherwise throw an error
+#' @export
+validate_model_options <- function(data, options) {
+  ## This must return TRUE if valid - otherwise throw an error
+  ## TODO: Add real validation
```

Comment:
:joy: is there a ticket you can point this at?

Reply:
I can add that - I'm expecting Jeff to add this as he discovers problems. Perhaps even a process during workshop as we learn what options cause problems. So I can add a task but I'm not expecting us to sit down and impl this as a whole feature in one go.

Reply:
I've added the ticket number to this